howto_conda_package.md

```diff
@@ -0,0 +1,361 @@
+```
+
+The installer will ask you to install the dependencies for your package like during any other conda tool installation process. 
+1. Check, the requirements. If all are there, continue. If not, you may check your meta.yaml. Most likely your `conda-build` process would break anyway.
+2. After the installation finished, you are free to test your conda environment. Yippi!
+
+### **Upload package**
+
+#### Upload to Anaconda Cloud
+
+This is the easiest way of distributing your package. An Upload to anaconda makes your package generally available under a channel you define by yourself. For convenience, this can be your anaconda username.
+
+1. Create an account at [anaconda](https://anaconda.org/)
+2. Login: In the dashboard you get an overview on packages, projects, notes and so on. For more information click on [upload a package](https://docs.anaconda.com/anaconda-cloud/user-guide/tasks/work-with-packages/#uploading-packages)
+
+Follow the instructions provided.
+
+#### Upload to Bioconda Recipe Repository
+
+Bioconda provides a documentation for contributing to Bioconda. You can find it [here](https://bioconda.github.io/contributor/index.html). For contributing to bioconda, you need a github account. If you do not have one already, now is the time.
+
+0. Be as sure as you can that your package and tool functions correctly. You can never withdraw from whats in the cloud! 
+1. Read the guidlines of bioconda on [this page](https://bioconda.github.io/contributor/guidelines.html). If something is missing, you might include it in your recipe.
+  Most important here is including a sha256 checksum from a stable URL. As stable bioconda considers a packed tarball file from a github release. On this, you can calculate the checksum using:
+```
+wget -O- <URL> | shasum -a 256
+```
+There are also other sources considered as stable, for more information, consider the guidlines mentioned before. 
+3. Bioconda recipes are stored in a github repository called "bioconda-recipes". For contribution, you are obliged to fork the bioconda-recipes: Login to your github account and click [here](https://github.com/bioconda/bioconda-recipes/fork) to create a remote fork which is a repository in your account.
+4. Now, that you have a copy of bioconda-recipes, create a local clone on your computer running:
+   ```
+      git clone https://github.com/<USERNAME>/bioconda-recipes.git
+   ```
+5. Set the bioconda-recipes repo as upstream branch to be tracked by your local branch. Doing so, your local branch keeps track with the actual bioconda repo. More about upstream branches: [here](https://devconnected.com/how-to-set-upstream-branch-on-git/).
+
+```
+cd bioconda-recipes
+git remote add upstream https://github.com/bioconda/bioconda-recipes.git
+```
+
+6. After pulling the latest changes in the bioconda repo, create a new branch with an appropriate name.
+```
+# Make sure our master is up to date with Bioconda
+git checkout master
+git pull upstream master

```

Comment:
Before you had written here also to 

```
git push origin master
```

but I think that's not necessary because you anyway don't want to do any changes to the master directly (and also you can't because its forked)

Reply:
This is from the bioconda doc: https://bioconda.github.io/contributor/workflow.html#id4
However, it seems not necessary...

Reply:
ah okay... when it is in the doc, we might also just use it. please add it again!